blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -24,12 +28,17 @@ categories:
 
 ---
 
-We are excited to share with you the open-source contributions of our team.
-We have been actively contributing to a number of open-source projects that have
-a significant impact on the industry. Through these contributions, we have been
-able to help improve the overall functionality and security of these projects,
-while also gaining valuable experience and knowledge that we can apply to our
-client work.
+From 2021 to 2022, our team of developers thrived, making significant
+contributions to firmware projects. Our primary focus revolved around
+[coreboot](https://3mdeb.com/training/#coreboot-for-embedded-linux-developers),
```

Comment:
Why are we linking to training? We can connect to the official coreboot page or tag coreboot in our blog post.

Linking to training does not make sense in this context.

Reply:
Each fix should be addressed in a separate commit; otherwise, realizing what is fixed for this particular issue and what is fixed for others is still problematic.

Please stop making bulk commits during the review.